docs/reference/cli.md

```diff
@@ -203,12 +203,13 @@ Options:
                             will fail.
   --auto-apply              Automatically apply the new plan after creation.
   --no-auto-categorization  Disable automatic change categorization.
-  --effective-from          The effective date from which to apply forward-only
-                            changes on production.
-  --include-unmodified      Whether to include unmodified models in the target
-                            development environment.
-  --model-selections        Model selection strings to filter the
-                            models that should be included in this plan.
+  --include-unmodified      Include unmodified models in the target
+                            environment.
+  --select-model TEXT       Select specific model changes that should be
+                            included in the plan.
+  --backfill-model TEXT     Backfill only the models whose names match the
```

Comment:
does this support special syntax?

Reply:
For example did we decide to not support a syntax for saying directly modified only?

Reply:
Depends on what you mean by a special syntax. If you mean `+`, `*`, etc. then yes. If you mean directly modified, then no. I realized supporting directly modified doesn't have much value.